src/instructlab/data/generator/generate_data.py

```diff
@@ -20,13 +20,9 @@
 import tqdm
```

Comment:
It would be helpful to not move any of the code under the `generator` directory. All of that is moving to the `instructlab.sdg` library.

Ideally it wouldn't be changed, either, other than what is absolutely necessary.

Reply:
For now, I think the sake of this design generator should live in `data` when sdg library comes around this dir can simply be deleted hopefully.